content/platform/minimise-costs.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "Minimise Costs"
+weight = 1
+chapter = false
+pre = ""
++++
+
+The platform allows you to optimise your infrastructure for maximum cost efficiency. In situations where you have multiple environments, some may be significantly underutilised.
```

Comment:
I think the logic for using Spot instances to solve the cost for underutilised environments is a bit confusing. 

If I have an underutilised environment then GCP will automatically downscale all my pools, so that I don't pay for them. 

OTOH if I need to run more workloads, but cheaper, then Spot instances make sense to me.